modules/apps/jaggery/emm/src/emm/units/header/header.hbs

```diff
@@ -76,7 +76,7 @@
         <div class="content">
             <div class="row">
                 <div class="col-lg-5 col-md-6 col-centered">
-                    <h3>Done. Password reset successful.</h3>
+                    <h3>Password change successful.</h3>
```

Comment:
Better to have a wording like "Password change is successful".

Reply:
Fixed